zio-cli/shared/src/main/scala/zio/cli/Options.scala

```diff
@@ -379,15 +367,18 @@ object Options extends OptionsPlatformSpecific {
               .validate(r._1, conf)
               .foldZIO(
                 _ => ZIO.succeed((r._1, Left(r._2))),
-                _ =>
+                _ => {
+                  // `uid` will only be None for Options.Empty. Which means the user would
+                  // have had to purposefully compose Options.Empty | otherArgument.
+                  val leftUid  = left.uid.getOrElse("???")
+                  val rightUid = right.uid.getOrElse("???")
```

Comment:
Another option would be to make `uid` non-optional, and give it a value of `"Empty"` or something for `Options.Empty`.